tools/flow9/backends/cpp/cpp.flow

```diff
@@ -126,13 +126,13 @@ void flow_println(A d) {
 	}
 }
 
-bstatement2cpp(backend : Backend<CppInfo>, s : BStatement) -> string {
+bstatement2cpp(backend : Backend<CppInfo>, s : BStatement, parsedType : string) -> string {
```

Comment:
I don't understand what "parsedType" means. Is this the expected type, or something else?